pkg/container/client.go

```diff
@@ -157,6 +157,15 @@ func (client dockerClient) GetContainer(containerID t.ContainerID) (Container, e
 		return Container{}, err
 	}
 
+	containerNetworkMode := strings.Split(string(containerInfo.HostConfig.NetworkMode), ":")
+	if len(containerNetworkMode) == 2 {
```

Comment:
```suggestion
	if len(containerNetworkMode) == 2 && containerNetworkMode[0] == "container" {
```
if this contains some other prefix that we don't know about, we should probably leave it alone